guide/payments/activity.md

```diff
@@ -0,0 +1,209 @@
+
+Closely linked to search is export, which can serve several purposes. For one, a user may want to migrate to a new wallet and take their transaction history with them. Second, it is useful for accounting and tax purposes, especially for merchants.
+
+{% include image-gallery.html pages = page.imagesSearch %}
+
+### Different views
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/activity/budgeting.png"
+   retina = "/assets/images/guide/payments/activity/budgeting@2x.png"
+   alt-text = ""
+   caption = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Alternative approaches to the basic list view can give users different perspectives on their finances and activity.
+
+The example shown here uses traditional categories borrowed from [personal household finance]({{ '/guide/designing-products/personal-finance/' | relative_url }}). As users tag transactions, the categories update.
+
+It's recommended to approach this type of view based on the unique use case and feature set of the application. For example, a wallet that is focused on interaction with Lightning applications may instead group payments by the services they were made with.
+
+</div>
+
+### Transaction details
+
+The activity list focuses on summarizing the top-level information, so users can quickly scan the screen to get an overview. If they identify a transaction they want to take a closer look at or interact with, the following screens become relevant.
+
+##### A payment received on-chain
+
+As with the activity list, transaction details screens should also only highlight relevant informationand options, and hide tertiary details behind extra taps.
+
+{% include image-gallery.html pages = page.imagesReceive %}
+
+##### A payment made on Lightning
+
+While the details for Lightning and on-chain payments look very similary, there are subtle differences. Most noticeable for the user is the difference in fees, as well as the fast processing time.
+
+{% include image-gallery.html pages = page.imagesSend %}
+
+### Status display
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/payments/activity/status-field.png"
+   retina = "/assets/images/guide/payments/activity/status-field@2x.png"
+   alt-text = ""
+   caption = ""
+   width = 250
+   height = 219
+   layout = "float-right-desktop -background -shadow"
+%}
+
+Status indicators are important for on-chain transactions, since they take longer to confirm, as well as when things go wrong. While a completed transactions generally doesn't need a status indicator, it should be shown if the transaction completes while the user is watching. Error messages should be clear and provide access to details and options to remedy the problem.
```

Comment:
```suggestion
Status indicators are important for on-chain transactions since they take longer to confirm, as well as when things go wrong. While a completed transaction generally doesn't need a status indicator, it should be shown if the transaction completes while the user is watching. Error messages should be clear and provide access to details and options to remedy the problem.
```